discovery-provider/src/queries/get_attestation.py

```diff
@@ -72,7 +69,7 @@ def is_valid_oracle(address: str) -> bool:
     if oracle_addresses:
         oracle_addresses = oracle_addresses.decode().split(",")
     else:
-        oracle_addresses = get_oracle_addresses_from_chain(redis)
+        oracle_addresses = [] # get_oracle_addresses_from_chain(redis)
```

Comment:
nit, can we comment here to turn this on when it's on chain

Reply:
ofc